tasks/install_shell.sh

```diff
@@ -315,6 +337,8 @@ do_wget() {
   if test $? -eq 0; then
     critical "ERROR 404"
     unable_to_retrieve_package
+  else
+    rc=retry_cmd "wget -O "$2" "$1" 2>$tmp_stderr"
```

Comment:
shouldn't `"` be escaped inside the string?
`"wget -O \"$2\" \"$1\" 2>$tmp_stderr"`

Reply:
might be simpler to use simple quotes:

`"wget -O '$2' '$1' 2>$tmp_stderr"`

Reply:
done